app/controllers/prior_authority/steps/start_page_controller.rb

```diff
@@ -8,15 +8,34 @@ def show
           view_context, application: current_application, show_index: false
         )
 
-        @tasklist = StartPage::TaskList.new(
-          view_context, application: current_application
-        )
+        @tasklist = start_page_tasklist
         # # passed in separately to current_application to
         # # allow it to be wrapped in a presenter in the future
         @application = current_application
         render 'laa_multi_step_forms/task_list/show',
                locals: { header: -> {}, skip_progress: true, app_type: 'application' }
       end
+
+      def start_page_tasklist
+        if further_information_needed
+          @tasklist || StartPage::FurtherInformationTaskList.new(
+            view_context, application: current_application
+          )
+        else
+          @tasklist || StartPage::TaskList.new(
+            view_context, application: current_application
+          )
+        end
+      end
+
+      def further_information_needed
+        if current_application.further_informations.empty?
+          false
+        else
+          last_further_info_request = current_application.further_informations.last.created_at
```

Comment:
I think we need to be explicit that we want the most recent further_information. So that would be: `further_informations.order(:created_at).last.created_at`, but we can do even better and just do `current_application.further_informations.maximum(:created_at)` and avoid loading a whole model object.